tests/unittests/core/experiment.yaml

```diff
@@ -469,6 +480,7 @@
   # orion's configuration
   pool_size: 2
   max_trials: 1000
+  working_dir: 
```

Comment:
Does it crash if `working_dir` is not in yaml?

Reply:
Well, `orion_config.yaml` doesn't have it and the tests pass fine ;)

Reply:
I mean in the DB most specifically ;)